babel.config.js

```diff
@@ -1,7 +1,13 @@
 // This is the configuration file for babel
 /* eslint-env node */
 module.exports = {
-    presets: ["@babel/preset-env", "@babel/preset-react"],
+    presets: [
+        ["@babel/preset-env", {
+            useBuiltIns: "entry",
```

Comment:
In this other ticket @leonexcc was able to use "usage", which allowed to not add imports to the code and worry less about keeping polyfills up-to-date. The compatibility there is based on browserlist which I think is preferred while we don't have a lot of time for Klaro's development.

https://github.com/KIProtect/klaro/pull/124/commits/885347481eac466dbc1d68a6d64028435d525c02

Reply:
How does this work? Should the user add polyfills by himself or are all required polifills added automatic?